CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ changes in the following format: PR #1234***
 - Session Current_stage default value changed for Not Started (PR #7102)
 - Fix public pages with missing title (PR #7121)
 - New data dictionary framework and module (#6936)
+- React Form Select Element now has the ability to set an option in the element as a disabled option. (PR #7306)
```

Comment:
this is a good `notes for developers` item, it can also live in updates and improvements (subsection of core below). and yes an item can be in both notes for developers and another section like core or modules